webidl/person.webidl

```diff
@@ -0,0 +1,6 @@
+
+dictionary Person {
+    required sequence<LocalizableString> name;
+             DOMString                   id;
```

Comment:
We need an example for this in the spec.

Reply:
Added in https://github.com/w3c/wpub/pull/285/commits/aefe40f27a55ebf37d4272c8d887d32d5bcd2bf3